frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.scss

```diff
@@ -23,3 +23,8 @@
 .pf-c-wizard__toggle {
   z-index: calc(var(--pf-global--ZIndex--xs) - 1);
 }
+
+// account for smaller console font size
```

Comment:
It would be nice if we could add this to the :before so we aren't adding extra spacing. 
Is there anyway to flag this so when the fonts get updated (assuming they will) we can pull this out. Otherwise we'll see the same misalignment with the new font.

Reply:
thanks, I applied this also to `pf-c-wizard__toggle-list-item` for the responsive view